bundle/edu.gemini.spdb.reports.collection/src/main/scala/edu/gemini/dbTools/ephemeris/NonSiderealTargetRef.scala

```diff
@@ -8,20 +8,20 @@ import edu.gemini.spModel.target.env.TargetEnvironment
 import edu.gemini.spModel.target.obsComp.TargetObsComp
 import edu.gemini.spModel.util.SPTreeUtil
 
-/** A reference to an individual non-sidereal observation. */
-case class NonSiderealObservation(oid: SPObservationID, hid: HorizonsDesignation, targetName: String)
+/** A reference to an individual non-sidereal target reference. */
+case class NonSiderealTargetRef(oid: SPObservationID, hid: HorizonsDesignation, targetName: String)
```

Comment:
Before it would find just the science target for the observation and be done.  Now it will look at all the targets, base position + guide targets and include all that are non-sidereal.  Non-sidereal target reference seemed more appropriate for that since each one no longer corresponds to a single observation.